src/Chainweb/Miner/POW.hs

```diff
@@ -131,9 +131,22 @@ powMiner lf conf nid cdb = runForever lf "POW Miner" $ do
                    (int . Seq.length $ _payloadWithOutputsTransactions payload)
                    (int bytes)
                    (estimatedHashes p newBh)
+            !adb = AmberdataBlock
+                   (_blockHeight newBh)
+                   (_blockHash newBh)
+                   (_blockCreationTime newBh)
+                   (_blockParent newBh)
+                   (_blockNonce newBh)
+                   (_blockMiner newBh)
+                   (_minedBlockSize nmb)
+                   (_minedBlockTrans nmb)
+                   (_blockChainId newBh)
+                   (_blockWeight newBh)
 
         logg Info $! "POW Miner: created new block" <> sshow i
         lf @(JsonLog NewMinedBlock) Info $ JsonLog nmb
+        logg Info $! "POW Miner: sent AmberData block" <> sshow i
+        lf @(JsonLog AmberdataBlock) Info $ JsonLog adb
```

Comment:
Am I reading this correctly that the miner is sending blocks to Amberdata?  If that's the case, then it seems like Amberdata won't receive blocks mined by any other miners?  That will work for now, but ultimately I think it would be better to have consensus send the blocks so that Amberdata will get all the blocks no matter who mines them.  @larskuhtz would it be hard to make this change?

Reply:
Moved to use cut monitoring, so logging all blocks (not just the ones mined by the current node).